test/NuGet.Core.FuncTests/Dotnet.Integration.Test/Dotnet.Integration.Test.csproj

```diff
@@ -3,10 +3,14 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   
   <PropertyGroup>
-    <TargetFrameworks>$(NETCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(NETCoreTargetFramework);netcoreapp3.0</TargetFrameworks>
```

Comment:
Use the property. 

Basically the idea is we don't mention the TFMs in the project files unless absolutely necessary.

Reply:
Thanks! Changed accordingly.